src/commandline_options.cpp

```diff
@@ -254,6 +254,7 @@ commandline_options::commandline_options(const std::vector<std::string>& args)
 		("log-precise", "shows the timestamps in log output with more precision.")
 		("no-log-to-file", "log output is written to standard error rather than to a file.")
 		("log-to-file", "log output is written to a file even if it would normally go to standard output/error. Cancels the effect of --no-log-to-file whether implicit or explicit.")
+		("no-con", "For Windows, when used with --no-log-to-file, results in output being written to cerr/cout instead of CONOUT. Otherwise, does nothing.")
```

Comment:
Why make up a new option instead of using --wnoconsole as before?

Reply:
I also don't understand why the option is not under a windows ifdef now.

Reply:
I think it's more descriptive, and IIRC there was a comment asking about having the options be consistent so that it doesn't error out if present on a non-windows OS.

Reply:
Also it'd be `--wnoredirect` using the old option name.

Reply:
No, `--wnoredirect` was about not logging to file. The console options where about attaching or allocating a console and then redirecting to CONOUT$.

Hard to see how it's more descriptive. Perhaps at least write console out?

Reply:
I don't feel particularly strongly about it. If you have a different name that you'd prefer, feel free to change it.

Reply:
Well, as mentioned I would stick to the original name.

Reply:
Updated.